conda/core/portability.py

```diff
@@ -48,25 +48,34 @@ class _PaddingError(Exception):
     pass
 
 
+def _subdir_is_win(subdir: str) -> bool:
+    if "-" in subdir:
+        os, _ = subdir.split("-")
```

Comment:
```suggestion
        os, _ = subdir.split("-", 1)
```

In case there are more than one dash in the subdir (there shouldn't be but we've seen custom subdirs before).

Reply:
```suggestion
        os, _ = subdir.lower().split("-", 1)
```